webui/react/package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "postshrinkwrap": "npx npm-force-resolutions && npm install --ignore-scripts",
-    "start": "craco start",
-    "build": "craco build",
+    "start": "vite serve",
```

Comment:
As mentioned in the engineering doc and meeting, vite intentionally doesn't support the stuff that would make the type checking in create react app work. I don't think this is a terrible loss because most modern editors are able to leverage the preexisting typescript language server to enable type checking. We do manually check types during the build just to be safe, which should cause the react build to fail during github ci.

just pushed an update to ensure the server starts on port 3000.

Reply:
i think its good to have type check. 
https://vitejs.dev/guide/features.html#transpile-only

>During development, if you need more than IDE hints, we recommend running tsc --noEmit --watch in a separate process, or use [vite-plugin-checker](https://github.com/fi3ework/vite-plugin-checker) if you prefer having type errors directly reported in the browser.